### PR TITLE
Add separete groups and users with all branch permissions

### PR DESCRIPTION
**Pull request by AndreySurzhan**
Add code which allows to create separate groups, users,
adding users into appropriate group, assigning appropriate branch
permissions for appropriate group
This commit allows to use this script in automation tests for branch
permission testing

Next Step: add group and user for CREATE_QUESTIONS permission when Q/A
feature is ready

### CODE REVIEW

**Comment by ctapobep** [comment]
Okay, @mixas27 looks like another PR was created, could you please review this one?

**Comment by ctapobep** [comment]
@mixas27 would you take a look?

**Comment by mixas27** [comment]
@ctapobep, code looks good. But query character register is not consistent.
@AndreySurzhan, Could you please correct  queries following next conventions: 
1. MySQL keywords (like `select`, `insert`, `set` etc.) should be lowercased
2. Names of tables and columns should be upercased

**Comment by AndreySurzhan** [comment]
@mixas27  and @ctapobep 
1. MySQL keywords (like select, insert, set etc.) should be lowercased - DONE
2. Names of tables and columns should be upercased - DONE

**Comment by ctapobep** [comment]
Coo, thanks. @mixas27 have a look and if everything is fine, please rebase it on master.

File: jcommune-model/src/main/resources/org/jtalks/jcommune/model/sample-forum.sql [hunk 1]
```diff
@@ -1,32 +1,67 @@
-﻿SET @forum_component_id := 2;
```
Comment (mixas27) [inline jcommune-model/src/main/resources/org/jtalks/jcommune/model/sample-forum.sql]: Why you removed this line? On newly created database component ID will be undefined.

File: jcommune-model/src/main/resources/org/jtalks/jcommune/model/sample-forum.sql [hunk 2]
```diff
@@ -1,32 +1,67 @@
...
-insert ignore into COMPONENTS (CMP_ID, COMPONENT_TYPE, UUID, `NAME`, DESCRIPTION) VALUES (2, 'FORUM', (SELECT UUID() FROM dual), 'JTalks Sample Forum', 'Available users: admin/admin registered/registered moderator/moderator banned/banned');
```
Comment (mixas27) [inline jcommune-model/src/main/resources/org/jtalks/jcommune/model/sample-forum.sql]: @forum_component_id shuld be used instead of "magic" number

**Comment by AndreySurzhan** [comment]
Hi @ctapobep  and @mixas27 
Add ``` -﻿SET @forum_component_id := 2 ```
Add ```@forum_component_id := 2``` instead  ```2``` into line №2
Please review.

**Comment by AndreySurzhan** [comment]
Can be closed  because of https://github.com/jtalks-org/jcommune/pull/30

**Comment by ctapobep** [comment]
You could've closed it yourself ;)

**Pull request closed without merging**